### DISCUSSION TITLE
Capital letters in channel name problem.

### DISCUSSION THREAD

**Discussion by BCC-BGChatCommunity** [qaisjp/go-discord-irc#117] [Q&A] [answered]
The all issue is [here](https://github.com/qaisjp/go-discord-irc/issues/115#issue-842591244) Anyone can help to resolve it ?

**Comment by qaisjp** [comment, marked as answer]
@Rapman4 suggested a nice fix in that issue. I have pushed that change, I hope it solves your problem!
